src/waveresponse/_transform.py

```diff
@@ -0,0 +1,162 @@
+import numpy as np
+
+from ._core import RAO
+
+
+def rigid_transform(
+    t: np.array, surge: RAO, sway: RAO, heave: RAO, roll: RAO, pitch: RAO, yaw: RAO
+):
+    """
+    Rigid body transformation of (surge, sway and heave) RAOs.
+
+    Transforms surge, sway and heave RAOs from one location to another by assuming
+    rigid body motion. Note that the rotational degrees-of-freedom (i.e., roll,
+    pitch and yaw) does not need transformation, since these are independent of
+    location, and thus will be the same for all points on the rigid body.
+
+    Parameters
+    ----------
+    t : array-like
+        Translation vector given as (x, y, z) coordinates. Determines the position
```

Comment:
Maybe use (u, v, w) to describe this translation vector, ref. comment on theory manual.

Reply:
I think I prefer to call it x, y, z coordinates in the function docstring to indicate better that this is cartesian coordinates?